NestJS/src/gmail_api/gmailapi.js

```diff
@@ -90,43 +104,91 @@ const getMessageById = async (messages,gmail,callback) => {
     const message = await singleMessage;
     msgList.push(message);
   }));
-  return getEmailAttachmentId(msgList);
+  return await getEmailAttachmentId(msgList,gmail);
```

Comment:
there is no need in await before return since the whole function is a asynced thus will return promise